index.js

```diff
@@ -2,25 +2,36 @@
 const { Structures, Invite } = require('discord.js');
 const fetch = require("node-fetch")
 
+
+
 Structures.extend('VoiceChannel', VoiceChannel => {
   return class EpicVoiceChannel extends VoiceChannel {
     constructor(client, data) {
       super(client, data);
+      this.applications = {
+        "youtube_together": "755600276941176913",
+        "fishington": "814288819477020702",
+        "chess_in_the_park": "832012774040141894",
+        "chess_in_the_park_dev": "832012586023256104",
+        "betrayal": "773336526917861400",
+        "poker_night": "755827207812677713"
+      }
     }
 
     /**
      * Creates Activity Invite in the voice channel
-     * @param {string} ApplicationID - Application ID
+     * @param {string} Application - Application
      * @returns {Invite}
      */
-    activityInvite(ApplicationID) {
+    activityInvite(application) {
+      if(!this.applications[application]) return;
       return new Promise(res => {
         let fetched = fetch(`https://discord.com/api/v8/channels/${this.id}/invites`, {
           method: 'POST',
           body: JSON.stringify({
             max_age: 86400,
             max_uses: 0,
-            target_application_id: ApplicationID,
+            target_application_id: this.applications[application],
```

Comment:
```suggestion
            target_application_id: this.applications[application]?this.applications[application]:application,
```
This is just for what if the application doesn't exists in our list